### ISSUE TITLE
No actionbar in Settings

### ISSUE THREAD

**Issue by cdhiraj40** [enviroCar/enviroCar-app#818]

**Comment by cdhiraj40** [comment]
https://user-images.githubusercontent.com/75211982/133747748-d594ead9-22b5-432e-ad0b-fecff07b049a.mp4

**Comment by cdhiraj40** [comment]
@asaikarthikeya I am done with work, I will make PR if you approve. Thanks

**Comment by asaikarthikeya** [comment]
Sure, go ahead! thanks.

**Comment by cdhiraj40** [comment]
> Sure, go ahead! thanks.

i will make it tomorrow. my laptop is getting debugged(repaired) :)

**Comment by SebaDro** [comment]
What's the thing with this issue? Would you please, provide a detailed description for the issue, in accordance to our [contributing guidelines](https://github.com/enviroCar/enviroCar-app/blob/master/CONTRIBUTING.md)

**Comment by asaikarthikeya** [comment]
> What's the thing with this issue? Would you please, provide a detailed description for the issue, in accordance to our [contributing guidelines](https://github.com/enviroCar/enviroCar-app/blob/master/CONTRIBUTING.md)

The back button on the top toolbar is missing in setting screen.

**Comment by SebaDro** [comment]
Yes, @dewall is right. If you have a closer look on the different views, you will notice that the toolbar layout is not consistent across the whole app. There are 3 different layouts:
1. enviroCar logo with back button
2. enviroCar logo without back button
3. text header with back button
In addition, there is another interesting design on the help view which switches the header layout using an animation.

Hence, we first should discuss which layout should be preferred for future releases.

**Comment by cdhiraj40** [comment]
Same 

> > What's the thing with this issue? Would you please, provide a detailed description for the issue, in accordance to our [contributing guidelines](https://github.com/enviroCar/enviroCar-app/blob/master/CONTRIBUTING.md)
> 
> The back button on the top toolbar is missing in setting screen.

The same issue is there for ``StartRecordingActivity``, any reason sir? @SebaDro @asaikarthikeya

**Comment by SebaDro** [comment]
Yes and no  :point_down:

> Yes, @dewall is right. If you have a closer look on the different views, you will notice that the toolbar layout is not consistent across the whole app. There are 3 different layouts:
> 
> 1. enviroCar logo with back button
> 2. enviroCar logo without back button
> 3. text header with back button
>    In addition, there is another interesting design on the help view which switches the header layout using an animation.
> 
> Hence, we first should discuss which layout should be preferred for future releases.

**Comment by cdhiraj40** [comment]
> Yes and no  :point_down:
> 
> > Yes, @dewall is right. If you have a closer look on the different views, you will notice that the toolbar layout is not consistent across the whole app. There are 3 different layouts:
> > 
> > 1. enviroCar logo with back button
> > 2. enviroCar logo without back button
> > 3. text header with back button
> >    In addition, there is another interesting design on the help view which switches the header layout using an animation.
> > 
> > Hence, we first should discuss which layout should be preferred for future releases.
> 
> 

yes i know that sir but that activity doesnt have any toolbar 

![Screenshot_2021-11-11-20-14-57-18_47898aacd586656d6f60f8abb18dee77.jpg](https://user-images.githubusercontent.com/75211982/141317738-3552943a-2705-4a25-a029-676beb985f3e.jpg)

**Comment by asaikarthikeya** [comment]
> > Yes and no  :point_down:
> > 
> > > Yes, @dewall is right. If you have a closer look on the different views, you will notice that the toolbar layout is not consistent across the whole app. There are 3 different layouts:
> > > 
> > > 1. enviroCar logo with back button
> > > 2. enviroCar logo without back button
> > > 3. text header with back button
> > >    In addition, there is another interesting design on the help view which switches the header layout using an animation.
> > > 
> > > Hence, we first should discuss which layout should be preferred for future releases.
> > 
> > 
> 
> yes i know that sir but that activity doesnt have any toolbar 
> 
> ![Screenshot_2021-11-11-20-14-57-18_47898aacd586656d6f60f8abb18dee77.jpg](https://user-images.githubusercontent.com/75211982/141317738-3552943a-2705-4a25-a029-676beb985f3e.jpg)
> 
> ![Screenshot_2021-11-11-20-18-16-72_47898aacd586656d6f60f8abb18dee77.jpg](https://user-images.githubusercontent.com/75211982/141317956-2237ae48-e658-4278-83ec-cb3e8a41520e.jpg)

Hello Dhiraj, Do you think a toolbar looks good in the recording screen? Toolbar is intentionally ignored in the recording screen. The back option is also not a must have feature in settings screen. Always try to think in the opposite way( who it will be if I have a toolbar, does it improve the UI?) before asking. Some things are designed intentional keeping several other parameters in mind. Thank you🙂

**Comment by cdhiraj40** [comment]
> > > Yes and no  point_down
> > > > Yes, @dewall is right. If you have a closer look on the different views, you will notice that the toolbar layout is not consistent across the whole app. There are 3 different layouts:
> > > > 
> > > > 1. enviroCar logo with back button
> > > > 2. enviroCar logo without back button
> > > > 3. text header with back button
> > > >    In addition, there is another interesting design on the help view which switches the header layout using an animation.
> > > > 
> > > > Hence, we first should discuss which layout should be preferred for future releases.
> > 
> > 
> > yes i know that sir but that activity doesnt have any toolbar
> > ![Screenshot_2021-11-11-20-14-57-18_47898aacd586656d6f60f8abb18dee77.jpg](https://user-images.githubusercontent.com/75211982/141317738-3552943a-2705-4a25-a029-676beb985f3e.jpg)
> > ![Screenshot_2021-11-11-20-18-16-72_47898aacd586656d6f60f8abb18dee77.jpg](https://user-images.githubusercontent.com/75211982/141317956-2237ae48-e658-4278-83ec-cb3e8a41520e.jpg)
> 
> Hello Dhiraj, Do you think a toolbar looks good in the recording screen? Toolbar is intentionally ignored in the recording screen. The back option is also not a must have feature in settings screen. Always try to think in the opposite way( who it will be if I have a toolbar, does it improve the UI?) before asking. Some things are designed intentional keeping several other parameters in mind. Thank youslightly_smiling_face

I meant the back option in recording screen activity, sorry for the inconvenience. I Will sure try to think that way next time before asking.

**Comment by pree-T** [comment]
Is this issue still open? May I work on it? Can someone help me?

**Comment by cdhiraj40** [comment]
> Is this issue still open? May I work on it? Can someone help me?

the issue is not open for work, there is an ongoing discussion for this.

**Comment by madhurmehta007** [comment]
@cdhiraj40 Is this issue still in discussion or I can work upon it ?
